docs/fundamentals/executor/executor-api.md

```diff
@@ -114,10 +117,13 @@ class MyExecutor(Executor):
         print(kwargs)
 
     @requests(on='/index')
-    def bar(self, **kwargs):
-        print(kwargs)
+    async def bar(self, **kwargs):
+        await asyncio.sleep(1.0)
+        print(f'Calling bar') 
 ```
 
+For I/O bound Executors it can be helpful to utilize Python `async def`. This means we can wait for multiple pending Executor function calls concurrently.
```

Comment:
This can be confusing I think, since most commonly, no concurrent code is run in Executors

Reply:
i actually don't know what is this section is about, I personally would never use the code snippet as show in that subsection.